fiftyone/core/annotation/generate_label_schemas.py

```
@@ -0,0 +1,431 @@
+import eta.core.utils as etau
+
+import fiftyone.core.annotation.constants as foac
+from fiftyone.core.annotation.validate_label_schemas import (
+    validate_label_schemas,
+)
+import fiftyone.core.annotation.utils as foau
+import fiftyone.core.fields as fof
+import fiftyone.core.labels as fol
+
+
+def generate_label_schemas(sample_collection, fields=None, scan_samples=True):
```

Ruff (0.14.8)

Boolean default positional argument in function definition

(FBT002)